Sources/Private/Model/DotLottie/DotLottieUtils.swift

```diff
@@ -52,7 +52,10 @@ extension FileManager {
 // MARK: - DotLottieError
 
 public enum DotLottieError: Error {
-  case invalidFileFormat
```

Comment:
Since this is public I think we need to preserve these enum cases, otherwise this would be a breaking change. Could we deprecate the existing cases instead of removing them?

Reply:
Haha oops accidentally merged the PR after I finished reviewing it. I posted a follow-up PR to add back these three public enum cases: https://github.com/airbnb/lottie-ios/pull/1970

Reply:
Agree, it makes sense to keep them until the next minor/major version is released.